src/scipp/__init__.py

```diff
@@ -54,39 +49,28 @@
 setattr(Variable, '__format__', _format_variable)
 del _format_variable
 
+from . import _binding
+from .compat.dict import from_dict, to_dict
+from .coords import show_graph, transform_coords
+# Mainly imported for docs
+from .core import Bins, Coords, GroupByDataArray, GroupByDataset, Masks, abs, acos, \
+    add, all, allclose, allsorted, any, arange, array, asin, atan, atan2, bin, bins, \
+    bins_like, broadcast, ceil, concat, cos, counts_to_density, cross, cumsum, \
+    datetime, datetimes, density_to_counts, divide, dot, empty, empty_like, epoch, \
+    equal, erf, erfc, exp, flatten, floor, floor_divide, fold, full, full_like, \
+    geomspace, greater, greater_equal, group, groupby, hist, identical, index, \
+    isclose, isfinite, isinf, islinspace, isnan, isneginf, isposinf, issorted, less, \
+    less_equal, linspace, log, log10, logical_and, logical_not, logical_or, \
+    logical_xor, logspace, lookup, matrices, matrix, max, mean, merge, midpoints, min, \
+    mod, multiply, nan_to_num, nanhist, nanmax, nanmean, nanmin, nansum, negative, \
+    norm, not_equal, ones, ones_like, pow, rebin, reciprocal, round, scalar, sin, \
+    sort, sqrt, squeeze, stddevs, subtract, sum, tan, to, to_unit, transpose, values, \
+    variances, vector, vectors, where, zeros, zeros_like  # NOQA
```

Comment:
Hmm, I don't feel this is a good change. Having the imports split by file gave us a chance to see if something was missing, or adding it easily. Should we ignore the `__init__.py`?

Reply:
Excluded `__init__.py` and `core/__init__.py` because both have a bunch of rather special imports.